linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/ignition/shared.yml

```diff
@@ -0,0 +1,15 @@
+        filesystem: root
+        mode: 0644
+        path: /etc/pki/tls/openssl.cnf
```

Comment:
Please check out the change that https://github.com/ComplianceAsCode/content/pull/5885 introduces - the filename is wrong.

Reply:
actually the contents of this file is correct and it's aligned with #5885 . It seems that they just copied /etc/pki/tls/openssl.cnf which contains by default the correct include: `opensslcnf.config` file.

Reply:
Ok, but it seems that it's including both `openssl.config` and `opensslcnf.config` files, the include with `openssl.config` needs to be removed.

Reply:
also there is something wrong with the encoded data, because after decoding it I can see things like this:
```
sh-4.4# less /etc/pki/tls/openssl.cnf
sh-4.4# grep '\.include\s* /etc/crypto-policies/back-ends/opensslcnf.config$' /etc/pki/tls/openssl.cnf
.include /etc/crypto-policies/back-ends/opensslcnf.config
sh-4.4# grep '\.include\s* /etc/crypto-policies/back-ends/opensslcnf.config$' /etc/pki/tls/openssl.cnf
.include /etc/crypto-policies/back-ends/opensslcnf.config
sh-4.4# grep '\.include\s* /etc/crypto-policies/back-ends/openssl.config$' /etc/pki/tls/openssl.cnf
sh-4.4# grep '\.include\s* /etc/crypto-policies/back-ends/opensslcnf.config$' /etc/pki/tls/openssl.cnf
.include /etc/crypto-policies/back-ends/opensslcnf.config
sh-4.4# less /etc/pki/tls/openssl.cnf
sh-4.4# less /etc/pki/tls/openssl.cnf
sh-4.4# cat /etc/pki/tls/openssl.cnf
```

And the example file appears twice in this content